src/vivarium_gates_mncnh/validation/formatting.py

```diff
@@ -0,0 +1,30 @@
+import pandas as pd
+from vivarium_testing_utils.automated_validation.constants import DRAW_INDEX, SEED_INDEX
+from vivarium_testing_utils.automated_validation.data_transformation import calculations
+from vivarium_testing_utils.automated_validation.data_transformation.formatting import (
+    TotalPopulationPersonTime,
+)
+
+
+class TotalLiveBirths(TotalPopulationPersonTime):
+    """Formatter for simulation data that contains total live births."""
+
+    def __init__(self, scenario_columns: list[str]) -> None:
+        self.measure = "live_births"
+        self.entity = "total"
+        self.raw_dataset_name = "births"
+        self.name = "total_live_births"
+        self.filter_value = "total"
+        self.filters = {"sub_entity": [self.filter_value]}
```

Comment:
If we add `{"pregnancy_outcome": ["live_birth"]}` to the filters, can we avoid having to override the `format_dataset()` method?